### ISSUE TITLE
Online event fired twice during connection on v0.6.0 library version

### ISSUE THREAD

**Issue by pc-magas** [xmppjs/xmpp.js#606]
I did this basic code in order to test the library:

```
const { client } = require('@xmpp/client');

process.env.NODE_TLS_REJECT_UNAUTHORIZED=0;

const initXmpp = function(xmpp) {

  xmpp.on('error', err => {
    console.error("Error occured",err.toString())
  });

  xmpp.on('offline', () => {
    console.log('🛈', 'offline');
  });

  xmpp.on('online', (address) => {
    console.log(address);
  });

  xmpp.once('open', function(el) {
    console.log(el);
  });

  xmpp.on('stanza', stanza => {
    console.log('⮈', stanza.toString())
  });

  xmpp.on('status', (status) => {
    console.log(status);
  });

  xmpp.on('stanza', stanza => {
    console.log('⮈', stanza.toString())
  });

  xmpp.start().then( value => {
    console.log("Value", value);
  }).catch( e =>{
    console.error("Error Occured",e)
  });
}

//Dummy local-only credentials
clientInstance=new client({
    'service':"xmpp://0.0.0.0:5222",
    'domain': "example.com",
    'username':"admin",
    'password':"admin",
});

initXmpp(clientInstance);

process.on('unhandledRejection', function (reason, p) {
  console.error('Possibly Unhandled Rejection at: Promise ', p, ' reason: ', reason);
})
```
And I notice that on connection using a local `openfire` xmpp server in a docker container the status `online` is updated twice on a successful connection, thus resulting the `online` to be fired twice. Do you know why that happens? Is this a bug?

**Comment by sonnyp** [comment]
> And I notice that on connection using a local openfire xmpp server in a docker container the status online is updated twice on a successful connection, thus resulting the online to be fired twice. Do you know why that happens? Is this a bug?

What makes you think the event is fired twice?

**Comment by pc-magas** [comment]
Here are the execution logs from the script mentioned above:

```
connecting
connect
opening
opening
opening
⮈ <iq type="result" id="78tzlb4i23g" to="0.0.0.0/1aq22aw3xi"><bind xmlns="urn:ietf:params:xml:ns:xmpp-bind"><jid>admin@0.0.0.0/1aq22aw3xi</jid></bind></iq>
⮈ <iq type="result" id="78tzlb4i23g" to="0.0.0.0/1aq22aw3xi"><bind xmlns="urn:ietf:params:xml:ns:xmpp-bind"><jid>admin@0.0.0.0/1aq22aw3xi</jid></bind></iq>
online
JID { _domain: '0.0.0.0', _local: 'admin', _resource: '1aq22aw3xi' }
Error Occured { TimeoutError
    at Promise.race.delay.then (/home/pcmagas/Kwdikas/master_thesis/custom_xmpp/node_modules/@xmpp/events/lib/timeout.js:10:13) name: 'TimeoutError' }
Error occured TimeoutError
online
JID { _domain: '0.0.0.0', _local: 'admin', _resource: '1aq22aw3xi' }
```

**Comment by sonnyp** [comment]
can you add

```
// Debug
xmpp.on('status', status => {
  console.debug('🛈', 'status', status)
})
xmpp.on('input', input => {
  console.debug('⮈', input)
})
xmpp.on('output', output => {
  console.debug('⮊', output)
})
```

and resend the log?

**Comment by sonnyp** [comment]
I have no idea what's going on :/ or how to reproduce, are you doing something more?
It *might* be fixed (or at least better) with 0.6.1, could you give it a try?

**Comment by pc-magas** [comment]
Yeap updating the lib had the issue solved.
